Makefile

```diff
@@ -1,21 +1,26 @@
-CHAIN_AG=agoricdev-6
+CHAIN_AG=agoricdev-8
 CHAIN_COSMOS=cosmoshub-testnet
-IMAGE_AGORIC=agoric/agoric-sdk:agoricdev-6
+IMAGE_AGORIC=agoric/agoric-sdk:agoricdev-8
 
-HERMES=docker run --rm -vhermes-home:/home/hermes:z -v$$PWD:/config hermes -c /config/hermes.config
+HERMES=docker run --rm -vhermes-home:/home/hermes:z -v$$PWD:/config hermes:0.9.0 -c /config/hermes.config
```

Comment:
why the `0.9.0` here? I think I have been using `0.11.x`

Reply:
we build the `hermes` image locally; surely we want to use whatever version we built, right?

Reply:
@dckc in `docker.compose.yml` 0.9.0 is specified. I reckon it wouldn't be nice to pin the version so that we the exact dependencies when debugging.
https://github.com/dckc/ibc-lab/pull/5/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R8

Reply:
I can update this TAG to `0.11.x` if you want?

Reply:
Consider the work we're doing on the `local-chain` branch. It uses `0.12.pre` (not `0.11.x`): https://github.com/dckc/ibc-lab/blob/local-chain/docker-compose.yml

I'm not sure how to reconcile these branches, but I think it would be easier if we did _not_ pin a version here.

who knows, though. this is all very rough.